data/parsers/utils.py

```diff
@@ -115,7 +115,8 @@ def add_cases(cases, toAdd, target, cols=default_cols):
                 if d[cols.index('time')] == time:
                     new = False
                     for i in range(1,len(d)):
-                        d[i] += e[i]
+                        if e[i]!=None and d[i]!=None:
+                            d[i] += e[i]
```

Comment:
Doesn't this lead to unexpected behaviour if d[i]= None? In that case, e[i] would not be used (although it probably should be replacing d[i]). How about 

```
if d[i] != None:
    if e[i]:
        d[i] += e[i]
elif e[i] != None:
    d[i] = e[i]
    
```

Reply:
But in general, I think this is a good observation and definitely should go into this function.

Reply:
I agree, I did the above changes.